pasqal_cloud/batch.py

```diff
@@ -79,16 +82,28 @@ def _load_configuration(
             conf_class = EmuTNConfig
         elif values["device_type"] == EmulatorType.EMU_FREE.value:
             conf_class = EmuFreeConfig
-
         return conf_class.from_dict(configuration)
 
     @root_validator(pre=True)
-    def _build_job_dict(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+    def _build_job_dict_and_list(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        """This root validator will modify the 'jobs' attribute (which is a list
```

Comment:
I added this docstring as it was not easy to understand how is working pydantic and root validators, easier to read for the next ones who will need to understand this as the 'jobs' transformation is something important to remember

Reply:
nice!